packages/components/addon/components/hds/alert/index.js

```diff
@@ -12,31 +21,6 @@ export default class HdsAlertIndexComponent extends Component {
     return this.args.icon ?? null;
   }
 
-  /**
-   * We need a title or description to exist
-   * @method Alert#entity
-   * @return {object} { title, description }
-   */
-  get entity() {
-    let entity = { title: this.args.title, description: this.args.description };
```

Comment:
the `entity` in the `IconTile` was used to differentiate two different types of tile, while here you're using to create super-prop that then is used to control the presence of `title/text`. there's a simpler way (see previous comment) about the _constructor_ method.